composer.json

```diff
@@ -27,7 +27,8 @@
 		"php" : "^5.6.0 || ^7.0 || ^8.0",
 		"paragonie/random_compat" : "2.0.20",
 		"yzalis/identicon" : "2.0.0",
-		"mlocati/ip-lib" : "1.14.0"
+		"mlocati/ip-lib" : "1.14.0",
+		"google/cloud-storage": "^1.23"
```

Comment:
Very minor: We use explicit, precide versions to avoid accidental upgrades that are overlooked when working on other dependencies and therefore not documented in the changelog. And it is always 100% clear what version got used at any point in time.

Reply:
done